googletest/include/gtest/gtest.h

```diff
@@ -760,6 +778,13 @@ class GTEST_API_ TestInfo {
   // Returns the result of the test.
   const TestResult* result() const { return &result_; }
 
+  // Returns the number of executed assertions that has passed silently:
+  size_t success_assert_count() const { return result_.success_assert_count(); }
+
+  // Increment the number of assertions executed without errors:
+  size_t increment_success_assert_count(size_t v = 1) {
```

Comment:
return void as before

Reply:
Please remove argument size_t v = 1, it is unused, therefore not needed for the internal-only API

Reply:
Should be private, this is for internal use

Reply:
We need the v argument in KTF, because the test has executed in the kernel, and the number of asserts in that test are reported together. Since we try to implement our KTF as a user of Googletest without hacks to get to internal APIs, we also need it to be public.